node/src/components/network/tasks.rs

```diff
@@ -210,11 +211,13 @@ impl<REv> NetworkContext<REv> {
         net_metrics: &Arc<Metrics>,
     ) -> Self {
         // Set the demand max from configuration, regarding `0` as "unlimited".
-        let max_in_flight_demands = if cfg.max_in_flight_demands == 0 {
-            usize::MAX
-        } else {
-            cfg.max_in_flight_demands as usize
-        };
+        let max_in_flight_demands = chain_info.networking_config.map(|cfg| {
```

Comment:
This needs to be renamed at the very least, I believe this is `buffer_size` these days?

We could also name this `outgoing_buffer_size` or `send_buffer_size`, although configs would have to be updated as well, just `buffer_size` is a bit ambiguous.

Reply:
This was to be removed.